test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -192,13 +189,22 @@ def reply_to_message_at_index(self, index: int, message: str):
         message_object_to_reply_to = self.get_message_at_index(index)
         verify(not is_null(message_object_to_reply_to), "Message to reply to is loaded")
         move_mouse_over_object(message_object_to_reply_to)
-        click_obj_by_name(ChatComponents.REPLY_TO_MESSAGE_BUTTON.value)
+        foundreplyToButton = getChildrenWithObjectName(message_object_to_reply_to, "replyToMessageButton")[0]
+        verify(not is_null(foundreplyToButton), "Reply button found")
+        move_mouse_over_object(foundreplyToButton)
+        time.sleep(1)
```

Comment:
Why do we have to wait 1 sec before clicking the reply button? Is the timer necessary?

One thing I suggested to Pascal while he was looking at something similar is to use the right-click menu instead, since we seem to have so many issues with the buttons that show up on hover. What do you think?

Reply:
Sleep removed. Seems to work ok.